commcare_connect/opportunity/management/commands/auto_approval_opportunities.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management import BaseCommand
+
+from commcare_connect.opportunity.models import Opportunity, OpportunityAccess
+from commcare_connect.opportunity.visit_import import update_payment_accrued
+
+
+class Command(BaseCommand):
+    help = "Run auto-approval logic over opportunity"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--opp", type=int, required=True, help="ID of the opportunity to run auto-approval logic on"
+        )
+
+    def handle(self, *args, **options):
+        opp_id = options["opp"]
+        try:
+            opportunity = Opportunity.objects.get(id=opp_id)
+            access_records = OpportunityAccess.objects.filter(opportunity=opportunity)
+            users = [access.user for access in access_records]
+
+            update_payment_accrued(opportunity=opportunity, users=users)
```

Comment:
The more I think about this, the more I think dont want to update `payment_accrued` just the status. In other words, I think just https://github.com/dimagi/commcare-connect/blob/main/commcare_connect/opportunity/visit_import.py#L158-L167

This also presents a good opportunity to break those out into some shared functions since much of that is duplicated in https://github.com/dimagi/commcare-connect/blob/main/commcare_connect/opportunity/tasks.py#L325

We can have both of those call the same utility functions (where behavior is shared), and have this call that function as well.

Reply:
[fb7d9c1]([fb7d9c1](https://github.com/dimagi/commcare-connect/pull/373/commits/fb7d9c1836a80f8bbaff5ea17e8d7b0616cd7bdf))
Added